dask_snowflake/core.py

```diff
@@ -175,14 +188,14 @@ def read_snowflake(
     label = "read-snowflake-"
     output_name = label + tokenize(
         query,
-        connection_kwargs,
+        #connection_kwargs,
```

Comment:
Yes, that is a TODO. How to tokenize when a connection is input rather than the kwargs to build a connection. I should have opened this PR as a draft first.